app/controllers/miq_policy_controller.rb

```diff
@@ -445,7 +436,7 @@ def replace_right_cell(options = {})
     replace_trees = Array(replace_trees)
     @explorer = true
 
-    trees = build_replaced_trees(replace_trees, %i[policy_profile policy condition action alert_profile alert])
```

Comment:
I guess the `policy_profile` doesn't belong in this one, but in the first...right?